### ISSUE TITLE
Unable to solve method calls for multiple inheritence if "super()" is used

### ISSUE THREAD

**Issue by mlangkabel** [CoatiSoftware/SourcetrailPythonIndexer#30]
Consider the following code snippet:
```python
class Foo:
  def foo(self):
    pass

class Bar:
  def bar(self):
    pass

class Baz(Foo, Bar):
  def baz(self):
    super().foo()
    super().bar()
```

Here the indexer is unable to solve the call to `super().bar()`.

**Comment by mlangkabel** [comment]
Reported this issue on the Jedi tracker: https://github.com/davidhalter/jedi/issues/1311

**Comment by mlangkabel** [comment]
Fixed by merge of pull request #47.
